src/plan_bot.py

```diff
@@ -99,6 +99,9 @@ def process_post(
                     print("topic mismatch: ", plan_id, post.id, plan_confidence)
 
                     posts_db.document(post.id).set(db_data)
+        else:
```

Comment:
I think this is at the wrong level of indent! This will save posts that don't even have `warrenplanbot` in them. 

I think what happened is that https://github.com/joegoldbeck/wpb-bot/commit/70044d5af2391ccf505c74ce946cff4d6b012c6e#diff-74c6bffa41796cd9ad0a847ab0a54422 changed the indentation on line 98.

It should be 

```
if match and not post.locked:
    # blah
elif not skip_tracking:
    print("topic mismatch: ", plan_id, post.id, plan_confidence)

    posts_db.document(post.id).set(db_data)
```

and then that should take care of the two cases where we want to save to the posts_db